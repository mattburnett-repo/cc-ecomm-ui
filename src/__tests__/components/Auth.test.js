import { render, screen } from '@testing-library/react';
import Auth from '../../components/auth/Auth';

describe('Auth', () => {
    test('it has an auth / log in component', () => {

    })
})
