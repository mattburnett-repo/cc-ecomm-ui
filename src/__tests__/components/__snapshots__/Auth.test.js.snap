// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders a snapshot renders a snapshot 1`] = `
<div>
  <div>
    <button
      aria-label="google-auth"
    >
      Log in with Google
    </button>
  </div>
  <div>
    <label
      htmlFor="username"
    >
      Username:
    </label>
    <input
      aria-label="username"
      id="username"
    />
  </div>
  <div>
    <label
      htmlFor="password"
    >
      Password:
    </label>
    <input
      aria-label="password"
      id="password"
      type="password"
    />
  </div>
  <div>
    <button
      aria-label="login"
    >
      Log in
    </button>
  </div>
</div>
`;
